.circleci/config.yml

```diff
@@ -436,7 +436,6 @@ jobs:
       docker_layer_caching: true
     environment:
       - MAGMA_ROOT=/home/circleci/project
-      - MAGMA_MODULES_FILE=/home/circleci/project/circleci/modules.yml
```

Comment:
The modules.yml files were used by the build.py script rather than the controller dockerfile

The refactorings to build.py make the modules.yml unneeded -- so e.g. for now building the controller image always uses the "default" set of modules (all of them)